src/actions_manager.coffee

```diff
@@ -67,11 +68,6 @@ define [
         ## TODO: THROW ERROR ON REJECT
         @reporter.sendBeacon(url, payload).then => cb and cb(@session.analytics_session)
```

Comment:
I suggest to return `@session` here: `cb(@session)` in order to have full access in session object in `productClick` action.

Reply:
What if we don't pass anything to the callback so that we can use the value if we want to signify an error later?

Reply:
It seemed a little arbitrary to return the session...